libs/ui/editor/src/lib/atoms/comment/commentHistory.tsx

```diff
@@ -27,6 +27,7 @@ function ChildComments({
   setRevision
 }: ChildCommentsProps) {
   if (!comments) {
+    // eslint-disable-next-line react/jsx-no-useless-fragment
```

Comment:
here too

Reply:
for fragments vs null - both approaches are okay and in the end the difference is marginal. But I feel that in places where we mean to return html, but we don't for some reason, it's better to return empty fragment than null as null is a different type than html element. 

In React with TypeScript, when deciding between returning null or an empty fragment (<></> or <React.Fragment></React.Fragment>) in the default case of a switch statement where you don't want to render anything, both approaches are technically correct, but they have subtle differences:

Returning null:

This is a common pattern in React for conditional rendering. When you return null, nothing gets rendered to the DOM in that part of the component tree. It signals that no component should be rendered.
It's more explicit in indicating that nothing should be rendered.
It can be more readable and clear, especially for developers who are new to React or reviewing your code.
Returning an empty fragment (<></> or <React.Fragment></React.Fragment>):

An empty fragment renders nothing to the DOM, similar to null. However, it maintains the component's presence in the component tree.
It can be useful if you are tracking the lifecycle of a component, as the component technically still exists even though it renders nothing.
It can also be useful if you anticipate that the component might need to render something in the future based on state changes or props.
Which one to choose? It largely depends on your specific use case and coding style preference. If you want to explicitly show that nothing should be rendered, use null. If you prefer keeping the component's presence or anticipate future changes, an empty fragment might be more suitable.

again, both approaches are okay and there's no bad answer. If you prefer for me to change it to null, I will.

Reply:
Here also whatever you and the rest of the Frontend devs think it's better :-)  But if we decide to leave it like this wouldn't it be clearer to disable the linting rule? Or do you think it gives us some extra value to explicit define "// eslint-disable-next-line react/jsx-no-useless-fragment"?
With the follwoing rule:
  "rules": {
    "react/jsx-no-useless-fragment": "off"
  }

Reply:
I would still keep the rule because often during development someone might omit that, For example. someone creates fragment, adds multiple elements inside, and then during development some stuff changes and he removes those multiple items inside fragment and just leaves one - it's easy to omit that there is still a fragment that shouldn't be there